docs/en_US/Compression/v2_pruning_algo.rst

```diff
@@ -200,6 +309,29 @@ User configuration for Activation APoZ Rank Pruner
 
 .. autoclass:: nni.algorithms.compression.v2.pytorch.pruning.ActivationAPoZRankPruner
 
+Performance Test
+^^^^^^^^^^^^^^^^
+
+.. list-table::
+   :header-rows: 1
+   :widths: auto
+
+   * - Model
+     - Accucacy
+     - Parameters
+     - FLOPs
+     - Sparsity
+   * - VGG-16
+     - 93.86%
+     - 14.98M
+     - 313.46M
+     - 
+   * - Pruned VGG-16
+     - 87.24%
+     - 0.64M
```

Comment:
why parameters are reduce so much with 80% sparsity?

Reply:
The pruning behavior may differ according to various default `mode` setting. It has three types: 'normal', 'global' and 'dependency_aware'. In the intermediate `config_list` we can see it does prune nearly 0.8 in each layer.